lighthouse-core/closure/typedefs/Config.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Typing externs file for config class.
+ * @externs
+ */
+
+/**
+ * @constructor
+ * @param{Object} config
+ */
+var Config = function(config) {};
```

Comment:
you can move the `Config` typedefs into the config/index.js file itself on each method